### PR TITLE
MARP-1481 Update dev.yml to refer ci.yml v5

### PR DESCRIPTION
**Pull request by tvtphuc-axonivy**

### CODE REVIEW

**Comment by tvtphuc-axonivy** [comment]
Hi @ivy-rew , I just updated the dev.yml to refer to ci.yml v5.

May I ask which ivyPluginVersion and testerVersion we should use? Or should we just keep the current ivyPluginVersion and testerVersion?

Thanks!

**Comment by ivy-rew** [comment]
yes sure: `13.1.0-SNAPSHOT` is the next available dev train which should be the default defined in these pipelines.
also the engineversion lower range should be set to `(12.0.0` ...

**Comment by tvtphuc-axonivy** [comment]
Hi @ivy-rew , I have just updated the default versions for ivyPluginVersion, testerVersion, and Divy.engine.version as per your advice. Could you please review them for me?

Many thanks!

**Review by nqhoan-axonivy — APPROVED** [review]
LGTM

File: .github/workflows/dev.yml [hunk 1]
```diff
@@ -12,19 +12,19 @@ on:
...
 
 jobs:
   build:
-    uses: axonivy-market/github-workflows/.github/workflows/ci.yml@v4
+    uses: axonivy-market/github-workflows/.github/workflows/ci.yml@v5
     with:
-      mvnArgs: '"-Divy.engine.download.url=https://dev.axonivy.com/permalink/${{ inputs.ivyVersion }}/axonivy-engine.zip" "-Divy.engine.version=(10.0.0,]" "-Dproject.build.plugin.version=${{ inputs.ivyPluginVersion }}" "-Dtester.version=${{ inputs.testerVersion }}" ${{ inputs.mvnArgs }}'
+      mvnArgs: '"-Divy.engine.download.url=https://dev.axonivy.com/permalink/${{ inputs.ivyVersion }}/axonivy-engine.zip" "-Divy.engine.version=(12.0.0,]" "-Dproject.build.plugin.version=${{ inputs.ivyPluginVersion }}" "-Dtester.version=${{ inputs.testerVersion }}" ${{ inputs.mvnArgs }}'
       javaVersion: 21
       mvnVersion: 3.9.8
```
Comment (ivy-rew) [inline .github/workflows/dev.yml:29]: in theory we can omit these params now, as they are already set to the same default values bi the referred ci.yml

**Review by ivy-rew — APPROVED** [review]
very nice, thanks

**Comment by ivy-rew** [comment]
a note on the tag. There's no need to create a new 'v6' tag. Just use the existing 'v5' tag, remove it and re-create it pointing to the now improved version after this merge.

**Pull request merged**